ttnn/cpp/ttnn/op_library/binary/binary_op.hpp

```diff
@@ -148,7 +148,7 @@ struct Binary {
         const float scalar,
         const std::optional<ttnn::MemoryConfig> &memory_config = std::nullopt,
         const std::optional<const DataType> &dtype = std::nullopt,
-        std::optional<std::vector<UnaryWithParam>> fused_activations = std::nullopt) {
+        std::optional<std::vector<std::string>> activations = std::nullopt) {
```

Comment:
Do we cary activation param in `string` or there are no params?

Reply:
@ayerofieiev-tt  The string is to make `UnaryWithParam` more user-friendly. And we don't carry any param in it.

For example, "gelu" is `UnaryWithParam(GELU, 1.0)` and we could add "more_accurate_gelu"  (or something like that) that would map to `UnaryWithParam(GELU, 0.0)`

And in case of something like `POW`, we would most likely just have a few hard-coded versions like "pow2" that would map to `UnaryWithParam(POW, 2.0)`but we didn't need it for now